src/main/java/org/isf/admission/service/AdmissionIoOperations.java

```diff
@@ -324,32 +324,87 @@ public boolean deletePatientPhoto(int patientId) throws OHServiceException {
 		return patientRepository.save(foundPatient) != null;
 	}
 	
+	/**
+	 * Returns the list of Admissions by pages
+	 *
+	 * @param dateFrom.
+	 * @param daTo
+	 * @param pageable
+	 * @return the list of {@link Admission}.
+	 * @throws OHServiceException if an error occurs during database request.
```

Comment:
There is no `throws` clause with the method signature; there are other occurrences of this same issue